core-ui/src/components/App/resourceSchemas/resourceSchemaWorkerApi.js

```diff
@@ -0,0 +1,45 @@
+export let schemasWorker = null;
```

Comment:
i'm not sure about exporting `schemasWorker` directly. Since we're doing an API here, I think it would be more consistent to export `terminate` and `isAvailable` functions instead (names can be different, of course)

Reply:
done